### ISSUE TITLE
Helper Idea : %C

### ISSUE THREAD

**Issue by RobinPoirot** [rodi01/RenameIt#33]
Hi, just wanted to add an idea that can help designers.
Have a namer "Helper" like %C that can give color hexadecimal.

(sorry if this isn't a real issue, and sorry for bad english).

**Comment by rodi01** [comment]
That's a good idea. I will see what it takes to do that.

**Comment by tockreativ** [comment]
Any word on what this would take? I'd be happy to help.

**Comment by rodi01** [comment]
@tockreativ I don't think it would take much to do it. I just don't have time to work on it at the moment. The fill API looks promising https://developer.sketch.com/reference/api/#fill
